librz/include/rz_analysis.h

```diff
@@ -1520,6 +1527,31 @@ RZ_API void rz_analysis_function_check_bp_use(RzAnalysisFunction *fcn);
 RZ_API void rz_analysis_update_analysis_range(RzAnalysis *analysis, ut64 addr, int size);
 RZ_API void rz_analysis_function_update_analysis(RzAnalysisFunction *fcn);
 
+/**
```

Comment:
Looks fine now, but please move the documentation from headers to implementation.